python/cuml/benchmark/algorithms.py

```diff
@@ -323,6 +335,51 @@ def all_algorithms():
             accepts_labels=True,
             accuracy_function=cuml.metrics.accuracy_score,
         ),
+        AlgorithmPair(
+            sklearn.svm.SVC,
```

Comment:
I'd like to have separate pairs for SVC-Linear and SVC-RBF (just differing in the kernel function they use) since performance can be very different for these two.